### PR TITLE
[Analytics Engine] Document new time-related SQL features

### PR DESCRIPTION
**Pull request by 46bit**
This PR adds documentation for a variety of new time-related SQL features in Analytics Engine. They're already live but mostly unannounced until this PR is merged.

New show statements:
  - `SHOW TIMEZONE`
  - `SHOW TIMEZONES`

New comparison operators:
  - `BETWEEN`

New functions:
  - `formatDateTime`
  - `toUnixTimestamp`
  - `toStartOfInterval`
  - `extract`

### CODE REVIEW

File: content/analytics/analytics-engine/sql-reference.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ meta:
 
 # Workers Analytics Engine SQL Reference
 
-## SHOW statement
+## `SHOW TABLES` statement
```
Comment (jimhawkridge) [inline content/analytics/analytics-engine/sql-reference.md]: If this is going to be backticked then SELECT statement should be too, below, for consistency.
ISTR that backticks in these titles don't render too well though (they are also rendered in the sidebar) so maybe better without?

File: content/analytics/analytics-engine/sql-reference.md [hunk 2]
```diff
@@ -576,6 +562,139 @@ Examples:
 SELECT format('blob1: {}', blob1) AS s FROM dataset;
 ```
 
+See also: [formatDateTime](#formatDateTime)
+
+### toDateTime
```
Comment (jimhawkridge) [inline content/analytics/analytics-engine/sql-reference.md:567]: Merge this with the one below?

File: content/analytics/analytics-engine/sql-reference.md [hunk 3]
```diff
@@ -576,6 +562,139 @@ Examples:
...
+```
+
+### formatDateTime
+
+Usage:
+```SQL
+formatDateTime(<datetime expression>, <format string>[, <timezone string>])
+```
+
+`formatDateTime` prints a datetime as a string according to a provided format string. The format
```
Comment (jimhawkridge) [inline content/analytics/analytics-engine/sql-reference.md]: Something got lost from the end of this?

File: content/analytics/analytics-engine/sql-reference.md [hunk 4]
```diff
@@ -576,6 +562,139 @@ Examples:
...
+extract(<time unit> from <datetime>)
+```
+
+`extract` returns an integer number of time units from a datetime. It supports
+`YEAR`, `MONTH`, `DAY`, `HOUR`, `MINUTE` and `SECOND`.
+
+Examples:
+```SQL
+-- extract the number of seconds from a timestamp (returns 15 in this example)
+extract(SECOND from toDateTime('2022-06-06 11:15:15'))
```
Comment (jimhawkridge) [inline content/analytics/analytics-engine/sql-reference.md]: For clarity, maybe??:
```suggestion
extract(SECOND from toDateTime('2022-06-06 11:30:15'))
```

**Review by jimhawkridge — CHANGES_REQUESTED** [review]

File: content/analytics/analytics-engine/sql-reference.md (hunk 1 above)
Comment (46bit) [reply content/analytics/analytics-engine/sql-reference.md]: Thanks, you're right, I'll fix

File: content/analytics/analytics-engine/sql-reference.md (hunk 4 above)
Comment (jimhawkridge) [reply content/analytics/analytics-engine/sql-reference.md]: Just noticed - I changed the wrong field in the above suggestion. Have edited it.

Comment (46bit) [reply content/analytics/analytics-engine/sql-reference.md]: Thanks :)

**Review by jimhawkridge — APPROVED** [review]
Nice :-)

**Review by angelampcosta — APPROVED** [review]

**Review by victor-perov — APPROVED** [review]

**Pull request merged**